lib/css/modules/_neverEndingReddit.scss

```diff
@@ -115,6 +115,10 @@
 #NREPause {
 	cursor: pointer;
 
+	.res-neverEndingReddit-hideFloatingPauseButton & {
```

Comment:
This should become a disabled boolean body class option, with a migration to enable it if found in a stylesheet module's option  body class value,  and filter out the row containing it.